frontend/src/routes/Video.tsx

```diff
@@ -233,7 +233,6 @@ type Props = {
 
 const VideoPage: React.FC<Props> = ({ eventRef, realmRef, basePath }) => {
     const { t } = useTranslation();
-    const rerender = useForceRerender();
```

Comment:
That change aint good. I put it here specifically to redraw more than just the player. It's also about the time/date information below the player (i.e. in the `<Metadata>` component). For the video block it's probably fine to do it only in the Player, but here we specifically need to rerender the whole player page.

Reply:
Makes sense. I pushed another commit.